crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -19,6 +19,29 @@ class Api::UpgradeController < ApiController
 
   api :GET, "/api/upgrade", "Show the Upgrade status object"
```

Comment:
just ignore it for now, `/api/upgrade` will change this sprint anyways, as soon as we start working on https://trello.com/c/OWehKX6K/126-5-implement-a-state-machine-for-the-upgrade